server/src/main/java/org/elasticsearch/index/query/TermsSetQueryBuilder.java

```diff
@@ -82,6 +85,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeGenericValue(values);
         out.writeOptionalString(minimumShouldMatchField);
         out.writeOptionalWriteable(minimumShouldMatchScript);
+        out.writeOptionalString(minimumShouldMatch);
```

Comment:
This needs to handle bwc serialization. 

There are multiple examples around the code base. 

Something like:
```
if (out.getTransportVersion().onOrAfter(TransportVersion.V_8_500_019)) {
```
^ Where you create the next transport version type in that class with a UUID